### PR TITLE
Adding Network Page and Virtual Networks Overview

### PR DESCRIPTION
**Pull request by mlunoe**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @pierlo-upitup

**Comment by kennyt** [comment]
this looks good to me. can't find anything wrong

**Comment by kennyt** [comment]
please run integration tests locally also

**Comment by mlunoe** [comment]
retest this please

File: src/js/pages/NetworkPage.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+    this.state = {};
+  }
+
+  componentWillMount() {
+    this.updateCurrentTab();
+  }
+
+  componentWillReceiveProps() {
+    this.updateCurrentTab();
```
Comment (rcorral) [inline src/js/pages/NetworkPage.js]: could we add `super` on these lifecycle methods?

**Comment by rcorral** [comment]
Looks pretty good! 🚢

**Pull request merged**